BackofficeBundle/Resources/public/js/backboneRouter.js

```diff
@@ -166,20 +166,17 @@ var OrchestraBORouter = Backbone.Router.extend({
 
   generateUrl: function(routeName, paramsObject)
   {
+    var optionalParam = /\(([^)]*):([^)]*)\)/g;
```

Comment:
So your regex would be more  linked to the pattern you have put `/\(([^)]*_):([^)]*)\)/g`